.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Bazelisk
+        run: |
+          wget https://github.com/bazelbuild/bazelisk/releases/download/v1.6.1/bazelisk-linux-amd64
+          echo "76c3af8cfe19f287b532990296bb58f02074d388856256b45a5f51bdbd196657 bazelisk-linux-amd64" | sha256sum --check
```

Comment:
In the checksum string, either two spaces or an asterisk before the file name. That it works now is just 'lucky'. :smile: 
https://askubuntu.com/a/1202214